index.html

```diff
@@ -1,15 +1,141 @@
 <!DOCTYPE html>
-<html >
+<html lang="en-US">
 	<head>
 		<meta charset="UTF-8">
 		<meta name="viewport" content="width=device-width, initial-scale=1.0">
-		<title></title>
-		<style >
-			
-		</style>
+		<title>Intuit. Signup</title>
 		<link rel="stylesheet" href="stylesheet.css" />
 	</head>
 	<body>
-		<h1>Intuit Login Page clone</h1>
+		<section>
+			<nav id="navbar">
+				<div class="link_container">
+					<div class="left_nav">
+						<div>Already have an Intuit account?</div>
+						<a href="#" class="link">I forgot my User ID or Password</a>
+					</div>
+					<div class="right_nav">
+						<button id="signin_button">
+						<span class="lock"></span>
+						Sign Up
+						</button>
+					</div>
+				</div>
+			</nav>
+			<header class="header">
+				<div class="title">
+					<a href="#"></a>
+				</div>
+			</header>
+			<section>
+				<div>
+					<header class="head">
+						<div>
+							<ul>
+								<li class="list_item list_icon" id="mint_icon"></li>
+								<li class="list_item list_text" id="mint_text"></li>
+								<li class="list_item list_icon" id="qb_icon"></li>
+								<li class="list_item list_text" id="qb_text"></li>
+								<li class="list_item list_icon" id="cb_icon"></li>
+								<li class="list_item list_text" id="cb_text"></li>
+							</ul>
+						</div>
+					</header>
+				</div>
+				<div class="main">
+					<section >
+						<header class="form_header">
+							<span class="form_title">Create an Intuit account</span>
+							<span class="form_title_des">
+								<span class="div">One account for everything Intuit, including Mint.
+									<a href="#" class="link">Learn more</a>
+								</span>
+							</span>
+						</header>
+						<form>
```

Comment:
#### 👍🏾

Nice job using `form` element, this makes the section represent a document section that contains interactive controls for submitting information to a web server. 👏🏾